### PR TITLE
Update docs for PodDisruptionBudget unhealthy pod eviction policy promotion to Beta

### PR DESCRIPTION
**Pull request by atiratree**
Add documentation for the beta promotion:
- https://github.com/kubernetes/enhancements/issues/3017
- https://github.com/kubernetes/kubernetes/pull/115363

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
Looks reasonable to me. @atiratree who can provide a tech review on this change / confirm that there are no relevant technical changes?

**Comment by atiratree** [comment]
^ @soltysh

File: content/en/docs/tasks/run-application/configure-pdb.md [hunk 1]
```diff
@@ -234,10 +234,10 @@ These pods are tracked via `.status.currentHealthy` field in the PDB status.
 
 ## Unhealthy Pod Eviction Policy
 
-{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="beta" >}}
 
 {{< note >}}
-In order to use this behavior, you must enable the `PDBUnhealthyPodEvictionPolicy`
+This feature is enabled by default. You can disable it by disabling the `PDBUnhealthyPodEvictionPolicy`
```
Comment (soltysh) [inline content/en/docs/tasks/run-application/configure-pdb.md:240]:
```suggestion
This feature is enabled by default. You can disable it by setting the `PDBUnhealthyPodEvictionPolicy=false`
```

**Review by soltysh — APPROVED** [review]
One nit, but lgtm from tech pov

File: content/en/docs/tasks/run-application/configure-pdb.md (hunk 1 above)
Comment (sftim) [reply content/en/docs/tasks/run-application/configure-pdb.md:240]: To me, “disabling the `PDBUnhealthyPodEvictionPolicy` feature gate” is idiomatic. I don't recommend making this change.

For example, if we one day allow managing API server feature gates with a configuration file, disabling a gate is actually more likely to look like:
```yaml
   …
   - PDBUnhealthyPodEvictionPolicy: false # colon, not equality symbol
   …
```

Comment (soltysh) [reply content/en/docs/tasks/run-application/configure-pdb.md:240]: That's even better suggestion :+1:

**Review by sftim — COMMENTED** [review]
/approve

Thanks

**Pull request merged**